src/popup/index.html

```diff
@@ -7,10 +7,11 @@
 		<link rel="stylesheet" href="/common/lib/emoji-mart-embed/dist/emoji-mart.css">
 		<link rel="stylesheet" href="index.css"> <!-- load later then emoji-mart so overwriting is possible -->
 
+		<script type="application/javascript" src="../browser-polyfill.js"></script>
 		<script defer src="../common/common.js" type="module" charset="utf-8"></script>
 		<script defer src="index.js" type="module" charset="utf-8"></script>
 		<!-- we need to load emoji-mart last, so autoFocus works -->
-		<script defer src="/common/lib/emoji-mart-embed/dist/emoji-mart.js"></script>
+		<script src="/common/lib/emoji-mart-embed/dist/emoji-mart.js"></script>
```

Comment:
Why no defer?

Reply:
It causes a race condition in Chrome, where the popup fails to load over half the time.